terraform/check_health.sh

```diff
@@ -12,11 +12,11 @@ do
     echo "$(date -u) Server available"
     exit 0
   else
-    if [ $fail_count -eq 11 ]; then
+    if [ $fail_count -eq 21 ]; then
```

Comment:
I bumped up the run count because it was intermittently failing but after running many times I've verified this works.